components/aws/sagemaker/tests/integration_tests/scripts/run_integration_tests

```diff
@@ -116,11 +116,17 @@ set -e
 function launch_eks() {
   echo "[Creating EKS] Launching EKS cluster $EKS_CLUSTER_NAME"
 
-  eksctl_args=( --managed --nodes "${EKS_NODE_COUNT}" --node-type=c5.xlarge --timeout=60m --region "${REGION}" --auto-kubeconfig --version "${EKS_CLUSTER_VERSION}" )
+  eksctl_args=( --managed --nodes "${EKS_NODE_COUNT}" --node-type=c5.xlarge --region "${REGION}" --version "${EKS_CLUSTER_VERSION}" )
   [ ! -z "${EKS_PUBLIC_SUBNETS}" ] && eksctl_args+=( --vpc-public-subnets="${EKS_PUBLIC_SUBNETS}" )
   [ ! -z "${EKS_PRIVATE_SUBNETS}" ] && eksctl_args+=( --vpc-private-subnets="${EKS_PRIVATE_SUBNETS}" )
 
-  eksctl create cluster "${EKS_CLUSTER_NAME}" "${eksctl_args[@]}" --enable-ssm
+  # This command can print out non yaml output too. Take caution when upgrading eksctl.
```

Comment:
this can make the upgrades brittle, can we create the cluster using a config file directly instead of using command line args to generate the config?

Reply:
Wouldn't the config file have to updated along with eksctl in this case, if the schema were to change.

Reply:
yes but schema is supported and easy to catch errors in future, also we can avoid going back and forth between file and command line args to create the cluster

fine to merge this for now since vpc-id is also required in the config file